### ISSUE TITLE
[BUG]: getHelper treats zero value as empty when set notAllowEmpty to true

### ISSUE THREAD

**Issue by palcoder** [phalcon/cphalcon#14556]
The function getHelper treats the zero value as empty when you set notAllowEmpty to true and return the default value.

to reproduce:
set a field (get or post) to zero value

```
echo $di->getRequest()->getQuery('status', 'trim', 'zero value', true);
```

**Comment by ruudboon** [comment]
What Phalcon version are you using?

**Comment by palcoder** [comment]
it is 4.0.0-rc.2

Version => 4.0.0-rc.2
Build Date => Nov  1 2019 19:51:35
Powered by Zephir => Version 0.12.10-fedc314

**Comment by ruudboon** [comment]
Is fixed in 4.0.x branch. Thnx @niden
